server/src/main/java/io/crate/statistics/TransportAnalyzeAction.java

```diff
@@ -96,15 +95,6 @@ public TransportAnalyzeAction(TransportService transportService,
         this.transportService = transportService;
         this.schemas = schemas;
         this.clusterService = clusterService;
-        transportService.registerRequestHandler(
-            INVOKE_ANALYZE,
```

Comment:
Is the `INVOKE_ANALYZE` now also unused?

Reply:
> Is the `INVOKE_ANALYZE` now also unused?

I think so, basically we introduced and removed AnalyzeRequest in a single PR - https://github.com/crate/crate/pull/9351

first https://github.com/crate/crate/pull/9351/commits/4ff54f9dc4e5c6a8a3e3827bf93a858bbeb55189#diff-bbb8f75278eefa1830ece4eb05eb338d06b935308e4fcd703d01fabadd7f7f95R53
and then after some refactoring https://github.com/crate/crate/pull/9351/commits/87fda2d0de933c4ac35abcac46159f5d68a641b9#diff-bbb8f75278eefa1830ece4eb05eb338d06b935308e4fcd703d01fabadd7f7f95R52

both `AnalyzeITest` (manual) and `TableStatsServiceIntegrationTest` (auto) passes - so I think it's unused indeed.